### PR TITLE
feat(VPagination): emit keydown event (#5028)

### PR DESCRIPTION
**Pull request by bludnic**
## Description
Emits keydown event for VPagination. Important: component should be focused to make it work.

## Motivation and Context
Solves #5028

## How Has This Been Tested?
unit | visually

## Markup:

<details>

```vue
<template>
  <v-container>
    <v-layout>
      <v-pagination
        v-model="page"
        tabindex="1"
        :length="length"
        @keydown="onKeydown"
      ></v-pagination>
    </v-layout>
  </v-container>
</template>

<script>
  import VPagination from '../src/components/VPagination'

  export default {
    components: { VPagination },
    data: () => ({
      page: 1,
      length: 6,
    }),
    methods: {
      onKeydown (e) {
        let page = this.page

        if (e.code === 'ArrowLeft') {
          page--
        } else if (e.code === 'ArrowRight') {
          page++
        }

        if (page > 0 && page <= this.length) this.page = page
      },
    },
  }
</script>
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [ ] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VPagination/VPagination.ts [hunk 1]
```diff
@@ -188,6 +188,9 @@ export default mixins(Colorable, Themeable).extend({
         value: this.onResize,
       }],
       class: this.classes,
+      on: {
```
Comment (jacekkarczmarczyk) [inline packages/vuetify/src/components/VPagination/VPagination.ts]: Pass all $listeners?

Comment (bludnic) [reply packages/vuetify/src/components/VPagination/VPagination.ts]: It makes sense

**Comment by nekosaur** [comment]
This is already possible using `.native` modifier?

**Comment by jacekkarczmarczyk** [comment]
We're passing all listeners in many other components as well

**Comment by johnleider** [comment]
I don't think this solves the referenced issue in its entirety. We should properly add a11y support instead of forcing the user to implement it by providing them the tools to do so. I'm also not against this PR as a nice feature, but it should remove #5028's reference.

**Comment by johnleider** [comment]
This Pull Request is being closed due to inactivity.

Thank you for your contribution and interest in improving Vuetify!

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).

**Pull request closed without merging**